model-desc/ins-model-props.yml

```diff
@@ -290,6 +290,7 @@ PropDefinitions:
   dataset_uuid:
     Desc: INS internal dataset id
     Type: string
+    Key: true
```

Comment:
It makes sense to add the `Key` property, though I notice that other nodes in the INS model don't use (or apparently require) them. If we add it here, we should add it elsewhere for consistency.